lib/java/test/org/apache/thrift/TestStruct.java

```diff
@@ -48,7 +38,23 @@
 import thrift.test.StructB;
 import thrift.test.Xtruct;
 
-public class TestStruct extends TestCase {
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+import java.nio.ByteBuffer;
+import java.util.HashMap;
+import java.util.Map;
```

Comment:
There seems to be an arbitrary reordering of imports based on your particular style. In general, it's good to minimize these kinds of personal style preferences changes in PRs to any project, as they can flip-flop repeatedly between styles with different contributors.

Ideally, the project would normalize its imports. I don't know how this can be done with Gradle, but I wrote a Maven plugin to do it: https://github.com/revelc/impsort-maven-plugin

Reply:
@ctubbsii i agree with you but let me try to file up a jira - along with code formatting. see https://issues.apache.org/jira/browse/THRIFT-5561